README.md

```diff
@@ -23,3 +26,79 @@ Supported by ROSIN - ROS-Industrial Quality-Assured Robot Software Components.
 More information: http://rosin-project.eu/
 
 This project has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement no. 732287.
+
+# ROS2
```

Comment:
Da das doch ein bisschen Text ist könnten wir das in eine separate md Datei stecken in docs oder so.